cmd/nodepool/create.go

```diff
@@ -1,6 +1,7 @@
 package nodepool
 
 import (
+	"github.com/openshift/hypershift/api/v1beta1"
```

Comment:
nit: Should this be `hyperv1 "github.com/openshift/hypershift/api/v1beta1"` to match cmd/cluster/core/create.go?